aces/Loris
# [User Accounts] Permissions panel fix

The permissions panel on the edit user page is not correctly displaying Permission descriptions for user's with superuser permission. This is because of a superuser permissions check which returned a list of permissions without a `label` parameter specified. 

View before:
<img width="1400" alt="Screen Shot 2021-05-14 at 2 49 21 PM" src="https://user-images.githubusercontent.com/35467361/118315502-98cb1300-b4c3-11eb-9416-bbd2716efa1a.png">

View after:
<img width="1436" alt="Screen Shot 2021-05-14 at 2 48 43 PM" src="https://user-images.githubusercontent.com/35467361/118315543-a6809880-b4c3-11eb-9e24-985626c4eb11.png">

#### Testing instructions (if applicable)
1. Go to edit a user as a superuser and check that the permissions are correctly displayed with their descriptions
2. Verify that the permissions are also displayed correctly for a user who is not a superuser

* Resolves #7437 
* Related to #6770
